### PR TITLE
Remove echonext host + Add etc/hosts infos

### PR DESCRIPTION
**Pull request by itkg-ngilain**

### CODE REVIEW

File: provisioning/hosts/group_vars/vagrant [hunk 1]
```diff
@@ -1,8 +1,20 @@
 ---
 user_root: vagrant
+
 path_front: /var/www/front-openorchestra
 
+hosts_localhost: 127.0.0.1 localhost openorchestra
```
Comment (nicolasThal) [inline provisioning/hosts/group_vars/vagrant]: Where are those parameters used?

**Comment by nicolasThal** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
@nicolasThal : update the inte vars

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**